db/migrate/20240206120211_add_users_name_null_constraint.rb

```diff
@@ -1,5 +1,5 @@
 class AddUsersNameNullConstraint < ActiveRecord::Migration[7.1]
   def change
-    change_column_null(:users, :name, false)
+    change_column_null(:users, :email, false)
```

Comment:
Hm, if old this code is already in master branch, you should create new migration for changing column, but it's ok for now)